client/src/components/ListItem/index.js

```diff
@@ -20,8 +21,26 @@ const useStyles = makeStyles((theme) => ({
     width: "min(800px, 80%)",
     marginTop: "1rem",
   },
+  button: {
+    background: "linear-gradient(45deg, #2196F3 30%, #21CBF3 90%)",
+    borderRadius: 3,
+    border: 0,
+    color: "white",
+    height: 40,
+    padding: "0 30px",
+    boxShadow: "0 3px 5px 2px rgb(0 121 255 / 30%)",
+  },
 }));
 
+const StyledText = styled.h1`
```

Comment:
I'm still not great at JS so I don't really know what this means.  Could you explain what this does?  More specifically, what I'm thinking is that "styled.h1'..." alters all h1 text.  Is this right?

Reply:
I believe so. I ended up just making styledText apart of our makeStyles using theme so it would be easier to edit colors down the line if we wanted to.